docs/src/test-components-js.md

```diff
@@ -135,7 +135,7 @@ import App from './App.vue';
 test.use({ viewport: { width: 500, height: 500 } });
 
 test('should work', async ({ mount }) => {
-  const component = await mount(<App />);
+  const component = await mount(App);
```

Comment:
I think that examples should always have embedded markup instead of passing component function, to be consistent everywhere.

Reply:
Thanks for the review, most comments have been resolved. I wonder if it's really better to use the `tsx` syntax for Vue. `tsx` in combination with Vue is rarely used _(not sure about testing)_. In addition, Svelte, Angular, and native web components will probably never get `tsx` support, making the `ts` api more prominent than the `tsx` api.

Reply:
Thanks for clarifying!